src/errors/runtime/templates.js

```diff
@@ -15,42 +15,41 @@ const DOCUMENTATION_LINKS = {
 export default {
     [RUNTIME_ERRORS.cannotCreateMultipleLiveModeRunners]:  'Cannot launch multiple live mode instances of the TestCafe test runner.',
     [RUNTIME_ERRORS.cannotRunLiveModeRunnerMultipleTimes]: 'Cannot launch the same live mode instance of the TestCafe test runner multiple times.',
-    [RUNTIME_ERRORS.browserDisconnected]:                  'The {userAgent} browser disconnected. If you didn\'t close the browser yourself, browser performance or network issues may be at fault.',
+    [RUNTIME_ERRORS.browserDisconnected]:                  'The {userAgent} browser disconnected. If you did not close the browser yourself, browser performance or network issues may be at fault.',
     [RUNTIME_ERRORS.cannotRunAgainstDisconnectedBrowsers]: 'The following browsers disconnected: {userAgents}. Cannot run further tests.',
     [RUNTIME_ERRORS.testRunRequestInDisconnectedBrowser]:  '"{browser}" disconnected during test execution.',
     [RUNTIME_ERRORS.cannotEstablishBrowserConnection]:     'Cannot establish one or more browser connections.',
     [RUNTIME_ERRORS.cannotFindBrowser]:                    'Cannot find the browser. "{browser}" is neither a known browser alias, nor a path to an executable file.',
     [RUNTIME_ERRORS.browserProviderNotFound]:              'Cannot find the "{providerName}" browser provider.',
-    [RUNTIME_ERRORS.browserNotSet]:                        "You haven't specified a browser.",
-    [RUNTIME_ERRORS.testFilesNotFound]:                    'No test files inside "{cwd}" match the specified patterns.\n' +
-                                                           'Check the patterns for errors:\n' +
+    [RUNTIME_ERRORS.browserNotSet]:                        "You have not specified a browser.",
+    [RUNTIME_ERRORS.testFilesNotFound]:                    'Could not find test files at the following location: "{cwd}".\n' +
```

Comment:
This error loooks wrong.
```js
ERROR Could not find test files at the following location: "./test123".
Check paths for errors:
C:\GitHub\testcafe-phoenix

or launch TestCafe from a different directory.
For more information on how to specify test locations, see https://testcafe.io/documentation/402639/reference/command-line-interface#file-pathglo
b-pattern.
```
1) It seems like we need to replace 
`throw new GeneralError(RUNTIME_ERRORS.testFilesNotFound, getConcatenatedValuesString(this.sources, '\n', ''), cwd);`
with 
`throw new GeneralError(RUNTIME_ERRORS.testFilesNotFound, cwd, getConcatenatedValuesString(this.sources, '\n', ''));`
[here ](https://github.com/DevExpress/testcafe/blob/master/src/runner/bootstrapper.ts#L195)
2) Now we have `Check paths for errors:` but there is can be a glob pattern (e.g. `"./test123/**/*.js"`) so let's leave `Check patterns for errors:`
3) Do we need the blank line before `or launch TestCafe from a different directory.` since now it's one sentence?

Please take a look @AndreyBelym

Reply:
1. Correct
2. Agree
3. This empty line should separate a list of patterns from the rest of the error message. Maybe we should add an empty line before the list? I think the message will have more consistent look this way.

Reply:
3. I have the same plan B for this. Agree.